src/index.js

```diff
@@ -33,40 +33,48 @@ class PlayerLoaderPlugin {
   }
 
   apply(compiler) {
-    compiler.hooks.emit.tapAsync('PlayerLoaderPlugin', (compilation, callback) => {
-      request.get(this.playerUrl).then((playerjs) => {
-        let assets = Object.keys(compilation.assets);
+    if (this.settings_.backwardCompatible) {
```

Comment:
It seems that this may be possible without an option by doing something like: 
```js
// for webpack > 3
if (compiler.hooks && compiler.hooks.emit) {
  compiler.hooks.emit.tapAsync('PlayerLoaderPlugin', this.downloadAndAppendPlayer.bind(this));
} else {
  compiler.plugin('emit', this.downloadAndAppendPlayer.bind(this));
}
```

Reply:
Such a coincidence! I was thinking of the same yesterday. I'll modify this

Reply:
@brandonocasey this is done.